VnE/Source/Extensions/ChemPack/voronoi.py

```
@@ -0,0 +1,418 @@
+        else:
+            ...
+
+        symms = [x[1] for x in sym_codes]
+        data = [tuple([x.atom_type, *list(x.cif_frac_coords)]) for x in atoms]
+        bools = [True if x in sel else False for x in atoms]
+
+        res = cpplib.VoronoiCalculation(cell, symms, data, bools, 6.0)
+
+        polyh = Polyhedra.fromCppLib(res, atoms, cell)
+
+        points_list = point_class.PointsList(TREE_MODEL.getRoot(), name=f'Voronoi polyhedron')
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)